tests/build/plugin-userscript.test.ts

```diff
@@ -0,0 +1,298 @@
+import fs from 'fs';
+
+// @ts-expect-error rewired
+import { __get__ } from '../../build/plugin-userscript';
+
+
+describe('git URLs', () => {
+    const GitURLs = __get__('GitURLs');
```

Comment:
Is this a work-around to not have to export the class because the export would only be needed by tests and is not expected to be imported from anywhere else?

Reply:
Exactly. Perhaps a bit overkill though.